pkg/client/config.go

```diff
@@ -548,7 +547,8 @@ func (img *Images) UnmarshalYAML(node *yaml.Node) (err error) {
 		case "webhookRegistry":
 			img.PrivateWebhookRegistry = v.Value
```

Comment:
should this also be depreciated?

Reply:
I didn't want to do that at this stage, because there might be situations when you want to get the traffic-manager from one place but the agents from another. But perhaps that's over ambitious.